plugins/inputs/pf/README.md

```diff
@@ -0,0 +1,65 @@
+# PF Plugin
+
+The pf plugin gathers counters from the FreeBSD/OpenBSD pf firewall.
+
+The pfstat command requires read access to the device file /dev/pf. You have several options to grant telegraf to run pfctl:
+
+* Run telegraf as root. This is strongly discouraged.
+* Change the ownership and permissions for /dev/pf such that the user telegraf runs at can read the /dev/pf device file. This is probably not that good of an idea either,
+* Configure sudo to grant telegraf to run pfctl as root. This is the most restrictive option, but require sudo setup.
+
+### Using sudo
+
+You may edit your sudo configuration with the following:
+
+```sudo
+telegraf ALL=(root) NOPASSWD: /sbin/pfctl -s info
+```
+
+### Configuration:
+
+```toml
+  # use sudo to run pfctl
+  use_sudo = false
+```
+
+### Measurements & Fields:
+
+
+- pf
+    - entries (integer, count)
```

Comment:
Need to add the other fields that are collected

Reply:
will update.